src/components/SharedComponents/TaxonResult.tsx

```diff
@@ -226,12 +229,21 @@ const TaxonResult = ( {
             white={white}
             isBackground={false}
           />
-          {!!( confidence && confidencePosition === "photo" ) && (
+          {!!( ( confidence || confidencePercentage ) && confidencePosition === "photo" ) && (
             <View className="absolute -bottom-4 w-full items-center">
-              <ConfidenceInterval
-                confidence={confidence}
-                activeColor={activeColor}
-              />
+              {confidencePercentage
```

Comment:
The only time we prop in `confidencePercentage` is with `confidencePosition === "text"` at the moment. So this here will never be shown. Can we remove it?